CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# [VuFind Developer Manual](https://vufind.org/wiki/development)
+
+## Prerequisites
+
+ - [Community Code of Conduct](https://github.com/vufind-org/vufind/CODE_OF_CONDUCT.md)
+ - [Coding Standards](https://vufind.org/wiki/development:coding_standards) - How to conform to VuFind's style.
+ - [Recommended Tools](https://vufind.org/wiki/development:recommended_tools) - Suggestions for successful VuFind development.
+ - [Making Pull Requests](https://vufind.org/wiki/development:making_pull_requests)
+ 
+## About VuFind
+
+ - [Architecture Notes](https://vufind.org/wiki/development:architecture) - Information on VuFind's design and dependencies.
+ - [How-Tos](https://vufind.org/wiki/development:howtos) - Step-by-step instructions on various VuFind development tasks.
+ - [Plugins](https://vufind.org/wiki/development:plugins) - Information on the types of plug-ins used to extend VuFind functionality.
+ - [Profiling](https://vufind.org/wiki/development:profiling) - Information on profiling VuFind's code to improve performance.
+ - [Testing](https://vufind.org/wiki/development:testing) - Notes on testing VuFind.
+ - [Troubleshooting](https://vufind.org/wiki/development:troubleshooting) - Notes on troubleshooting and debugging VuFind.
+ - [APIs](https://vufind.org/wiki/development:apis) - APIs that VuFind provides for searching and other functionality.
+ 
+## VuFind Planning
+ - [Bi-weekly developers call](https://vufind.org/wiki/developers_call)
+ - [GitHub pull requests](https://github.com/vufind-org/vufind/pulls) - A place to view and discuss work in progress on new features.
+ - [JIRA](http://vufind.org/jira) - VuFind's issue tracker.
+ - [Slack](http://vufind.slack.com/) - Email crhallberg@gmail.com for an invitation to the Slack if you cannot sign up directly.
+
+## Other Notes
+ - [Working with MARC](http://wiki.code4lib.org/index.php/Working_with_MaRC) - This Code4Lib page offers useful resources for dealing with MARC records.
```

Comment:
I think this is of pretty limited usefulness; not sure if it's worth including here.